### PR TITLE
[MINOR][DOCS] Fixing log message for better clarity. 

### PR DESCRIPTION
**Pull request by akshatb1**
Fixing log message for better clarity.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/SparkConf.scala [hunk 1]
```diff
@@ -568,7 +568,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     // If spark.executor.heartbeatInterval bigger than spark.network.timeout,
     // it will almost always cause ExecutorLostFailure. See SPARK-22754.
     require(executorTimeoutThresholdMs > executorHeartbeatIntervalMs, "The value of " +
-      s"${networkTimeout}=${executorTimeoutThresholdMs}ms must be no less than the value of " +
+      s"${networkTimeout}=${executorTimeoutThresholdMs}ms must be greater than the value of " +
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/SparkConf.scala:571]: Isn't it virtually same?

Comment (akshatb1) [reply core/src/main/scala/org/apache/spark/SparkConf.scala:571]: "no less" than is slightly confusing as the application fails if both values are same, hence changing it to greater. Please let me know if it makes sense.

Comment (HyukjinKwon) [reply core/src/main/scala/org/apache/spark/SparkConf.scala:571]: Okay. it's probably because English isn't my first language. cc @srowen.

Comment (srowen) [reply core/src/main/scala/org/apache/spark/SparkConf.scala:571]: "no less" would be "greater than or equal to" and the check does look for "greater than" so the correction is OK.

**Comment by akshatb1** [comment]
> Is it possible to make this value property driven?)

These commits are already merged to master (with property driven), not sure why they are showing up in this PR, overall diff is only log fix.

File: core/src/main/scala/org/apache/spark/SparkConf.scala (hunk 1 above)
Comment (akshatb1) [reply core/src/main/scala/org/apache/spark/SparkConf.scala:571]: @srowen Thanks for checking it out, yes, that was the intent. Could you please help in closing this PR?

**Comment by srowen** [comment]
@akshatb1 weird, probably because you made a merge commit. I usually just rebase onto master and force-push if necessary, but you needn't usually rebase.

**Comment by srowen** [comment]
Jenkins test this please

**Comment by srowen** [comment]
Merged to master/3.0

**Comment by akshatb1** [comment]
Thanks, @srowen for the help in merging this PR.

**Pull request closed without merging**